### PR TITLE
Remove address field

### LINKED ISSUE #31: Remove address field

### PR DESCRIPTION
**Pull request by tanjiaxian99**
Has been merged with the latest version of dev, so tags have been removed as well.

Closes #31

### CODE REVIEW

File: src/test/java/seedu/address/logic/parser/AddCommandParserTest.java [hunk 1]
```diff
@@ -41,76 +36,86 @@ public void parse_allFieldsPresent_success() {
...
         // multiple emails - last email accepted
-        assertParseSuccess(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_AMY + EMAIL_DESC_BOB
-                + ADDRESS_DESC_BOB, new AddCommand(expectedPerson));
+        assertParseSuccess(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_AMY + EMAIL_DESC_BOB,
+                new AddCommand(expectedPerson));
 
         // multiple addresses - last address accepted
-        assertParseSuccess(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB + ADDRESS_DESC_AMY
-                + ADDRESS_DESC_BOB, new AddCommand(expectedPerson));
+        assertParseSuccess(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB,
```
Comment (junwei26) [inline src/test/java/seedu/address/logic/parser/AddCommandParserTest.java:55]: I think we can delete this testcase since we no longer have addresses.

**Review by junwei26 — CHANGES_REQUESTED** [review]
LGTM, should be good once we remove that one redundant test case.

File: src/test/java/seedu/address/logic/parser/AddCommandParserTest.java (hunk 1 above)
Comment (tanjiaxian99) [reply src/test/java/seedu/address/logic/parser/AddCommandParserTest.java:55]: Yups thanks for pointing that out!

**Review by junwei26 — APPROVED** [review]
LGTM!

**Pull request merged**